docs/ruleset_functions/is_zone_mechanically_heated_and_not_cooled.md

```diff
@@ -0,0 +1,43 @@
+# is_zone_mechanically_heated_and_not_cooled
+**Schema Version:** 0.0.23  
+
+**Description:** determines whether a zone is mechanically heated, but not cooled.  Checks for transfer air
+
+**Inputs:**
+- **RMI**
+- **zone**
+
+**Returns:**  
+- **result**: a boolean - TRUE or FALSE
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- set a boolean has_heating_system to false: `has_heating_system = FALSE`
+- set a boolean has_cooling_system to false: `has_cooling_system = FALSE`
+- set result to FALSE: `result = FALSE`
+- get the hvac systems serving the zone by using the get_list_hvac_systems_associated_with_zone function: `list_hvac_systems = get_list_hvac_systems_associated_with_zone(RMI,zone)`
+	- loop through each HeatingVentilatingAirconditionsSystem system: `for system in list_hvac_systems:`
+		- check for a heating system: `if system.heating_system != NULL or system.heating_system.type != NONE:`
+			- set the boolean has_heating_system to true: `has_heating_system = TRUE`
+		- look for a cooling system: `if system.cooling_system != NULL or system.cooling_system != NONE:`
+			- there is a cooling system: `has_cooling_system = TRUE`
+- at this point, if there is no cooling system, continue looking in the zone terminals for a cooling system (if there is a cooling system at this level, the function should return false regardless of what is in the zone terminals, and we can skip to the end): `if !has_cooling_system:`
+	- look at each terminal in the zone: `for terminal in zone.terminals:`
```

Comment:
since we're looping through each HVAC system, there can be only one terminal for each zone, but there can be multiple zones for each HVAC system. so maybe line 27 should instead be 'for each zone served by the HVAC system' instead of 'for each terminal in the zone'?